include/netlink/route/link/bonding.h

```diff
@@ -12,6 +12,9 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
+extern void rtnl_link_set_bondmode(struct rtnl_link *, uint8_t);
+
+extern void rtnl_link_set_bondactiveslave(struct rtnl_link *, int);
```

Comment:
I think the name is not right. Compare to `rtnl_link_bridge_set_vlan_filtering()` which is bridge-specifc, and the name makes that clear. `rtnl_link_set_bondmode()` looks like `rtnl_link_set_carrier()`, which operates on the general link object.

I think it should be named `rtnl_link_bond_set_*()`.

Also, please move the new setters to the bottom, not before `rtnl_link_bond_alloc()`.